master/buildbot/config.py

```diff
@@ -816,6 +816,34 @@ def __init__(self, name=None, slavename=None, startSlavenames=None, slavenames=N
 
         self.description = description
 
+        self.customBuildUrls = customBuildUrls or {}
+        self._validateCustomBuildUrls()
+
+    def _validateCustomBuildUrls(self):
+        """
+        Validates that customBuildUrls  contains strings in the format {'name': 'url'}
```

Comment:
Should be: Validates that customBuildUrls is a dictionary containing only strings and in the format {'name': 'url'} (same goes for the error message)